ilc/spec/wrapWithCache.spec.js

```diff
@@ -0,0 +1,177 @@
+import chai from 'chai';
+import sinon from 'sinon';
+import wrapWithCache from '../common/wrapWithCache';
+
+describe('wrapWithCache', () => {
```

Comment:
You haven't tested concurrent data request with cold cache

Reply:
Updated